core/domain/exp_services.py

```diff
@@ -2557,6 +2564,28 @@ def revert_exploration(
 
     revert_version_history(exploration_id, current_version, revert_to_version)
 
+    models_to_put: List[
+        base_models.BaseModel
+    ] = []
```

Comment:
Why do we need to initialize an empty array?

Reply:
Also try to add a meaningful name to the variable

Reply:
Removed it.
This is the list of the new translation and opportunity models to be stored in the datastore. After initializing we are first extending the list to store new translation models and then extending to store new opportunity model. And then at the end we are using `put_multi()` function to store these models.
Also, would naming it `translation_and_opportunity_models_to_put` be a good idea?

Reply:
Sounds good

Reply:
Done with it.